base/operators.jl

```diff
@@ -328,9 +328,31 @@ eltype(::Type{Any}) = Any
 eltype(t::DataType) = eltype(supertype(t))
 eltype(x) = eltype(typeof(x))
 
-# function pipelining
+# function pipelining, composition & negation
+
 |>(x, f) = f(x)
 
+"""
+    ∘(f, g)
+    f ∘ g
```

Comment:
That's a little unfortunate though. Infix operators should allow an infix form. Even typing `?<:` in the REPL gives help for `<:(T1, T2)` - have you ever seen a single line of Julia code written like that??